src/main/java/seedu/address/logic/commands/AddBidCommand.java

```diff
@@ -0,0 +1,54 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_CLIENT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_MONEY;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_PROPERTY_ID;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.bid.Bid;
+
+public class AddBidCommand extends Command {
+
+
+
+    public static final String COMMAND_WORD = "add-bid";
```

Comment:
Should this be "add -bid" with a space in between so that it's more standardised? Likewise for "list -bid".

Reply:
ok will edit accordingly

Reply:
oh reason why i left out the white space is that the parser is looking for one word but adding the space counts it as 2 word and cannot detect it as a add bid command